### PR TITLE
[zh-cn] sync logging system-logs

### PR DESCRIPTION
**Pull request by my-git9**
content/zh-cn/docs/concepts/cluster-administration/logging.md
content/zh-cn/docs/concepts/cluster-administration/system-logs.md

### CODE REVIEW

File: content/zh-cn/docs/concepts/cluster-administration/logging.md [hunk 1]
```diff
@@ -180,16 +180,32 @@ You can configure two kubelet [configuration settings](/docs/reference/config-ap
...
+In order to perform an efficient log rotation in clusters where the volume of the logs generated by
+the workload is large, kubelet also provides a mechanism to tune how the logs are rotated in
+terms of how many concurrent log rotations can be performed and the interval at which the logs are
+monitored and rotated as required.
+You can configure two kubelet [configuration settings](/docs/reference/config-api/kubelet-config.v1beta1/),
+`containerLogMaxWorkers` and `containerLogMonitorInterval` using the
+[kubelet configuration file](/docs/tasks/administer-cluster/kubelet-config-file/).
+-->
+为了在工作负载生成的日志量较大的集群中执行高效的日志轮转，kubelet
+还提供了一种机制，根据可以执行多少并发日志轮换以及根据需要监控和轮换日志的间隔来调整日志的轮换方式。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/cluster-administration/logging.md:199]:
```suggestion
还提供了一种机制，基于可以执行多少并发日志轮换以及监控和轮换日志所需要的间隔来调整日志的轮换方式。
```

File: content/zh-cn/docs/concepts/cluster-administration/logging.md [hunk 2]
```diff
@@ -180,16 +180,32 @@ You can configure two kubelet [configuration settings](/docs/reference/config-ap
...
+<!--
+In order to perform an efficient log rotation in clusters where the volume of the logs generated by
+the workload is large, kubelet also provides a mechanism to tune how the logs are rotated in
+terms of how many concurrent log rotations can be performed and the interval at which the logs are
+monitored and rotated as required.
+You can configure two kubelet [configuration settings](/docs/reference/config-api/kubelet-config.v1beta1/),
+`containerLogMaxWorkers` and `containerLogMonitorInterval` using the
+[kubelet configuration file](/docs/tasks/administer-cluster/kubelet-config-file/).
+-->
+为了在工作负载生成的日志量较大的集群中执行高效的日志轮转，kubelet
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/cluster-administration/logging.md:198]:
```suggestion
为了在工作负载生成的日志量较大的集群中执行高效的日志轮换，kubelet
```

File: content/zh-cn/docs/concepts/cluster-administration/logging.md [hunk 3]
```diff
@@ -180,16 +180,32 @@ You can configure two kubelet [configuration settings](/docs/reference/config-ap
...
+还提供了一种机制，根据可以执行多少并发日志轮换以及根据需要监控和轮换日志的间隔来调整日志的轮换方式。
+你可以使用 [kubelet 配置文件](/zh-cn/docs/reference/config-api/kubelet-config.v1beta1/)
+配置两个 kubelet [配置设置](/zh-cn/docs/tasks/administer-cluster/kubelet-config-file/)：
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/cluster-administration/logging.md:201]:
```suggestion
配置两个 kubelet [配置选项](/zh-cn/docs/tasks/administer-cluster/kubelet-config-file/)：
```

File: content/zh-cn/docs/concepts/cluster-administration/logging.md [hunk 4]
```diff
@@ -312,15 +328,44 @@ For Kubernetes cluster components that run in pods, these write to files inside
...
+需要注意的是，默认位置 `/var/log/pods` 已使用很长一段时间，并且某些进程可能会隐式使用此路径。
+因此，更改此参数必须谨慎，并自行承担风险。
+
+<!--
+Another caveat to keep in mind is that the kubelet supports the location being on the same
+disk as `/var`. Otherwise, if the logs are on a separate filesystem from `/var`,
+then the kubelet will not track that filesystem's usage, potentially leading to issues if
+it fills up.
+-->
+另一个需要记住的警告是 kubelet 支持与 `/var` 位于同一磁盘上的位置。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/cluster-administration/logging.md:359]:
```suggestion
另一个需要留意的问题是 kubelet 支持日志写入位置与 `/var` 位于同一磁盘上。
```

File: content/zh-cn/docs/concepts/cluster-administration/logging.md [hunk 5]
```diff
@@ -312,15 +328,44 @@ For Kubernetes cluster components that run in pods, these write to files inside
...
+否则，如果日志位于与 `/var` 不同的文件系统上，那么 kubelet
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/cluster-administration/logging.md:360]:
```suggestion
否则，如果日志位于与 `/var` 不同的文件系统上，kubelet
```

File: content/zh-cn/docs/concepts/cluster-administration/logging.md [hunk 6]
```diff
@@ -312,15 +328,44 @@ For Kubernetes cluster components that run in pods, these write to files inside
...
+将不会跟踪该文件系统的使用情况，如果文件系统已满，则可能会出现问题。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/cluster-administration/logging.md:361]:
```suggestion
将不会跟踪该文件系统的使用情况。如果文件系统已满，则可能会出现问题。
```

**Review by ydFu — APPROVED** [review]
/approve 
@my-git9 Thanks

**Pull request merged**